.travis.yml

```diff
@@ -50,7 +50,15 @@ matrix:
       env:
         - JOBCACHE=8
 
-    - name: publish documentation
+    - name: Lint Python code with flake8
```

Comment:
Please include an environment variable which makes the cache unique. The other jobs use JOBCACHE for this, I think it is good to use JOBCACHE=<unique number here> in this case.

Reply:
env: JOBCACHE=9